kwmserver/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG CODE_VERSION=0.15.1
+FROM kopano/kwmserverd:${CODE_VERSION}
+ARG CODE_VERSION
+ENV CODE_VERSION="${CODE_VERSION}"
+
+USER root
+
+ENV DOCKERIZE_VERSION v0.6.1
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+COPY wrapper.sh /usr/local/bin
```

Comment:
Cant this use https://hub.docker.com/r/kopano/kwmserverd ?

Reply:
I thought I needed more changes, but by now have also removed dockerize. 
I'd like to keep the wrapper.sh script to easily run kwmserverd in insecure mode for demo environments, though. (which I do not really want to do with the `command` value in the compose file.

Reply:
On second though now that I changed the variable assignment in wrapper.sh I think we could drop this if this variable assignment would be done in https://hub.docker.com/r/kopano/kwmserverd directly. should I adapt the entrypoint script or do you want to?

Reply:
Go ahead :+1: